pyqtgraph/graphicsItems/LegendItem.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, size=None, offset=None, horSpacing=25, verSpacing=0,
         """
         GraphicsWidget.__init__(self)
         GraphicsWidgetAnchor.__init__(self)
-        self.setFlag(self.GraphicsItemFlag.ItemIgnoresTransformations)
+        # self.setFlag(self.GraphicsItemFlag.ItemIgnoresTransformations)
```

Comment:
Hi @leo603222 

Can you either delete this line, or undo the comment and make the removal of this line part of a separate PR?

Thanks!

Reply:
Sorry, I'm not familiar with git.
I have undo the change of LegendItem, and rename my branch name too.